app/src/main/res/values/string-untranslated.xml

```diff
@@ -31,9 +31,9 @@
     <string name="searchWidgetLabel">Search Widget</string>
 
     <!-- Treat returning users differently Experiment -->
-    <string name="returningUsersNewUserButton">I\'m new!</string>
-    <string name="returningUsersReturningUserButton">I\'ve been here before.</string>
-    <string name="returningUsersOnboardingDaxText"><![CDATA[The Internet can be kinda creepy.<br/><br/>Not to worry! Searching and browsing privately is easier than you think.<br/><br/>Let\'s get started 👍]]></string>
```

Comment:
We don't have anymore a different Dax text. It's now the same in control and both variants.